recipes/eternafold/bld.bat

```diff
@@ -0,0 +1,13 @@
+REM Move to conda-specific src directory location
+cd %SRC_DIR%\src
+
+:: Build Eternafold
+REM make CXX=%CXX% CXXFLAGS=-std=c++11 -O3 -mfpmath=sse -msse -msse2 -msse3 -DEVIDENCE_SR -DEVIDENCE_PARS -DNDEBUG -pipe -Wundef -Winline --param large-function-growth=100000 -Wall
+make CXX=%CXX% CXXFLAGS="-std=c++14 -O2 -mfpmath=sse -msse -msse2 -msse3 -DEVIDENCE_SR -DEVIDENCE_PARS -DNDEBUG -Wall"
```

Comment:
Right now conda doesn't have a method for checking whether the end user's machine support sse3. Use at your own risk.

Also, I think we already set some variables in CXXFLAGS, the environment variable via an activation script in our compiler package. If you are overiding these flags, it may prevent your program from compiling on our infrastructure.